### PR TITLE
memcache.lib: Update serializers to handle bytestring values

### PR DESCRIPTION
**Pull request by bsimpson63**
👓 @spladug @suzie-su

### CODE REVIEW

**Comment by suzie-su** [comment]
what the version of baseplate gonna be if this one got merged?

**Comment by spladug** [comment]
LGTM. Just waiting on CI.

**Comment by spladug** [comment]
When merged, this will be included in the upcoming 1.0 release. If you want this for the 0.30 series as well we'll need to backport it. A pull request against the `v0.30.x` branch would set that up.

**Comment by bsimpson63** [comment]
@suzie-su What version of baseplate are you using in subreddit service? Do you need this change backported to 0.30.x?

File: tests/unit/clients/memcache_tests.py [hunk 1]
```diff
@@ -261,6 +277,10 @@ def test_deserialize_str(self):
         value = memcache_lib.decompress_and_unpickle(key="key", serialized="val", flags=0)
         self.assertEqual(value, "val")
 
+    def test_deserialize_str(self):
```
Comment (spladug) [inline tests/unit/clients/memcache_tests.py]: this is the same name as the function above

```
./tests/unit/clients/memcache_tests.py:280:5: F811 redefinition of unused 'test_deserialize_str' from line 276
```

**Comment by bsimpson63** [comment]
💇‍♂

**Comment by suzie-su** [comment]
0.30.0 is what I am using

**Pull request merged**